### PR TITLE
Add napari-bio-sample-data recipe

### PR DESCRIPTION
**Pull request by goanpeca**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- ~[ ] If static libraries are linked in, the license of the static library is packaged.~
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by goanpeca** [comment]
We are in the process of creating conda packages for [napari plugins](https://www.napari-hub.org/) and related dependencies. Once this recipe passes all the checks, every time a new release goes into PyPI, a PR will be automatically open, with usually very minor need for changes so the process of maintaining this moving forward is small. Also the `@conda-forge/napari` team will be available to help as needed.

You can read a bit more about conda-forge at https://conda-forge.org/#contribute

--- 

@jaimergp, @chili-chiu do you agree with being added as maintainers?

**Comment by goanpeca** [comment]
Haven't heard from the author in a few weeks. Moving to review. The author can be added as maintainer in a subsequent PR.

**Comment by goanpeca** [comment]
This one is ready for review @conda-forge/staged-recipes :)

File: recipes/napari-bio-sample-data/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    - dask-core
+    - fsspec
+    - numpy
+    - python >=3.8
+    - s3fs
+    - zarr >=2.12.0
+
+test:
+  imports:
+    - napari_bio_sample_data
```
Comment (jaimergp) [inline recipes/napari-bio-sample-data/meta.yaml:32]:
```suggestion
    - napari_bio_sample_data
    - napari_bio_sample_data._sample_data
```

File: recipes/napari-bio-sample-data/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    - pip list
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/chili-chiu/napari-bio-sample-data
+  summary: a sample data plugin for bio-related demos
+  license: BSD-3-Clause
+  license_file: LICENSE
```
Comment (jaimergp) [inline recipes/napari-bio-sample-data/meta.yaml:43]:
```suggestion
  license_file: 
    - LICENSE
    - README.md  # credits for bundled images
```

File: recipes/napari-bio-sample-data/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  commands:
+    - pip list
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/chili-chiu/napari-bio-sample-data
+  summary: a sample data plugin for bio-related demos
+  license: BSD-3-Clause
```
Comment (jaimergp) [inline recipes/napari-bio-sample-data/meta.yaml:42]:
```suggestion
  license: BSD-3-Clause AND CC-PDDC AND LicenseRef-AllenInstitute AND LicenseRef-CellTrackingChallenge AND CC-BY-4.0
```

File: recipes/napari-bio-sample-data/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    - pip
+    - python >=3.8
+  run:
+    - aiobotocore >=1.0.1
+    - dask-core
+    - fsspec
+    - numpy
+    - python >=3.8
+    - s3fs
+    - zarr >=2.12.0
```
Comment (jaimergp) [inline recipes/napari-bio-sample-data/meta.yaml:28]:
```suggestion
    - zarr >=2.12.0
    - napari
    - scikit-image
```

**Pull request merged**